multicluster/test/e2e/framework.go

```diff
@@ -0,0 +1,388 @@
+package e2e
```

Comment:
Is there any plan to unify this file with `antrea.io/test/e2e/framework` in the future? I understand that it will be quite some work because for multi-cluster case we need clients for more than one cluster. However, having a entirely new framework and TestData struct means that for each resource (e.g. Service and ANP, CG in the future) we need to redefine CRUD functions for those, which is a lot of code duplication

Reply:
Thanks for yang's comment. I have already considered it. According to Bas's comment, I think we can use a Base library for both antrea framework and multicluster framework. But I think it's much code to factor, maybe we can do  it later ?